_posts/0000-01-02-d-hell.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
![image](https://user-images.githubusercontent.com/90872252/148405698-0cfcc443-422d-4669-ba39-ed3cccdf1b7a.png)

Reply:
Added meme to slidedeck

Reply:
Good meme